src/lib/forms/core/PriceField.js

```diff
@@ -131,18 +130,21 @@ export class PriceField extends Component {
       : null;
   };
 
-  renderCurrencyLabel = () => {
+  renderCurrencyLabel = props => {
     const {
       fieldPath,
       defaultCurrency,
       canSelectCurrency,
       required,
     } = this.props;
     const name = `${fieldPath}.currency`;
-    return canSelectCurrency ? (
-      <Field name={name} required={required} component={PriceDropdown} />
-    ) : (
-      <Label name={name}>{defaultCurrency}</Label>
+    return (
+      <Field
```

Comment:
nit: If I understood correctly from your issue description when we have a fixed currency the dropdown will have only one value. I see that [dropdown](https://react.semantic-ui.com/modules/dropdown/) supports also `disabled` prop, we could add that in case we have a fixed currency.

Reply:
The thing is, if the field is optional and the user wants to leave the field empty they must select the <kbd>-</kbd> option, otherwise this value will be sent to & refused by the server.

Reply:
I think it is a compromise that we can accept, at least until we have anything better.
Just to understand:
* by default, when creating a new record (empty form), currency fields will be with `-`, so the user will have to choose something if required and/or price has a value.
* when editing, the currency field has a value. Only when clearing the price value (which is rare I guess), then the user will have to set `-` in the currency field.
Is this correct?